analyzer/codechecker_analyzer/analyzer.py

```diff
@@ -131,7 +129,7 @@ def __get_statistics_data(args, manager):
     return statistics_data
 
 
-def perform_analysis(args, skip_handler, context, actions, metadata):
+def perform_analysis(args, skip_handler, context, actions, metadata_tool):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [perform_analysis is too complex (22)](https://app.codacy.com/app/CodeChecker/codechecker/pullRequest?prid=4375788)